Sources/KeyboardSupport/UIView+KeyboardSupport.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  UIView+KeyboardSupport.swift
+//  KeyboardSupport
+//
+//  Created by John Davis on 12/3/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    
+    /// highest level superview of the view hierarchy
+    var topLevelContainer: UIView {
+        var returnView = self
+        
+        while returnView.superview != nil {
+            returnView = returnView.superview!
+        }
+        
+        return returnView
+    }
+    
+    /// Walks the view hierachy starting with itself to build an array of non-hidden,
+    /// UITextInput subviews that can become first responder.
+    ///
+    /// - Returns: Array of UITextInput subviews
+    func getAllTextInputViews() -> [UITextInputView] {
```

Comment:
Minor and somewhat personal preference, but "getAllTextInputViews" sounds very Java-like to me. Making this a property called simply `textInputViews` might feel a bit more Swifty.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/87024424dfb00a4495e26306032656b4571605fa